geometry_msgs/Pose[] pose_list
